scripts/elysium.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+let fullName = "Lekan Adetunmbi";
+let ID = "HNG-00221";
```

Comment:
change 'ID' to 'id' for this variable to and update the same for your output pass variable. rest all looks good.